crates/red_knot_python_semantic/resources/mdtest/annotations/tuple.md

```diff
@@ -0,0 +1,15 @@
+# Tuple
+
+## `Never`
+
+If a tuple type contains a `Never` element, then it is eagerly simplified to `Never` which means
+that a tuple type containing `Never` is disjoint from any other tuple type.
+
+```py
+from typing_extensions import Never
+
+def _(x: tuple[Never], y: tuple[int, Never], z: tuple[Never, int]):
+    reveal_type(x)  # revealed: Never
+    reveal_type(y)  # revealed: Never
+    reveal_type(z)  # revealed: Never
+```
```

Comment:
We have existing tests for this in `type_properties/tuples_containing_never.md`. Can we merge these new tests into the existing test suite, in case they increase coverage?

Reply:
https://github.com/astral-sh/ruff/pull/17137